.vscode/settings.json

```diff
@@ -4,11 +4,15 @@
   },
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
+  "editor.rulers": [100],
+  "editor.tabSize": 2,
```

Comment:
Prettier enforces a tab size of 2, and this codebase uses Prettier as the standard, so this needs to be reflected in the VSCode config as well so that it matches the project.